morelesszen_starterkit/js/morelesszen_starterkit.js

```diff
@@ -32,6 +32,31 @@ Drupal.behaviors.mobilemenu.attach = function(context, settings) {
   }
 };
 
+Drupal.behaviors.showMore = {};
+Drupal.behaviors.showMore.attach = function(context, settings) {
+  // link for revealing more info
+  $('.show-more', context).each(function(){
+    var $toggle = $(this);
+    var $target = $($toggle.attr('href'));
+
+    if ($toggle.is(':visible') && $target.length) {
+      $target.hide();
+      $toggle.on('click', function(e) {
+        $toggle.hide();
+        $target.slideDown();
+        e.preventDefault();
+      });
+    }
+  });  
+  // class for smooth scrolling to anchor
+  $('a.scroll', context).each(function(){
+    $(this).on('click', function(e){
```

Comment:
I think it’s possible to bind one single event handler to all links in the context instead of one event handler per link.

```js
$('a.scroll', context).on('click', function(e) {
  // …
});
```